[2022-06-08 22:34:33,353] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task 2022-06-08T22:30:04.509480+00:00 [queued]>
[2022-06-08 22:34:33,496] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task 2022-06-08T22:30:04.509480+00:00 [queued]>
[2022-06-08 22:34:33,497] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 22:34:33,497] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2022-06-08 22:34:33,502] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 22:34:33,536] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): get_name_task> on 2022-06-08T22:30:04.509480+00:00
[2022-06-08 22:34:33,557] {standard_task_runner.py:52} INFO - Started process 2902 to run task
[2022-06-08 22:34:33,566] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'get_name_task', '2022-06-08T22:30:04.509480+00:00', '--job-id', '63', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmpd1b56tea', '--error-file', '/tmp/tmp36lv41ch']
[2022-06-08 22:34:33,566] {standard_task_runner.py:77} INFO - Job 63: Subtask get_name_task
[2022-06-08 22:34:33,765] {logging_mixin.py:104} INFO - Running <TaskInstance: say_hi.get_name_task 2022-06-08T22:30:04.509480+00:00 [running]> on host 69873a4b1233
[2022-06-08 22:34:34,036] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=get_name_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-08T22:30:04.509480+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-08T22:30:04.509480+00:00
[2022-06-08 22:34:34,042] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-06-08 22:34:34,044] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo Alma > /opt/***/dags/username.txt']
[2022-06-08 22:34:34,058] {subprocess.py:75} INFO - Output:
[2022-06-08 22:34:34,069] {subprocess.py:83} INFO - Command exited with return code 0
[2022-06-08 22:34:34,171] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=get_name_task, execution_date=20220608T223004, start_date=20220608T223433, end_date=20220608T223434
[2022-06-08 22:34:34,247] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-08 22:34:34,314] {local_task_job.py:151} INFO - Task exited with return code 0
